Core.Collectors.Tests/Auditing/NoopAuditLogger.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.Cloud.InstrumentationFramework;
+using Microsoft.CloudMine.Core.Auditing;
+using Microsoft.CloudMine.Core.Telemetry;
+
+namespace Microsoft.CloudMine.Core.Collectors.Tests.Auditing
+{
+    public class NoopAuditLogger : IAuditLogger
```

Comment:
>NoopAuditLogger [](http://example.com/codeflow?start=17&length=15)

This is not used as far as I can tell.

Reply:
It is used in all the ADO Collector Tests wherever we need to pass AuditLogger